ios/REAModule.mm

```diff
@@ -43,7 +42,8 @@ - (void)_tryAndHandleError:(dispatch_block_t)block;
 @implementation REAModule {
 #ifdef RCT_NEW_ARCH_ENABLED
   __weak RCTSurfacePresenter *_surfacePresenter;
-  std::shared_ptr<NewestShadowNodesRegistry> newestShadowNodesRegistry;
+  std::shared_ptr<PropsRegistry> propsRegistry_;
+  std::shared_ptr<ReanimatedCommitHook> commitHook_;
```

Comment:
Do we need to retain commitHook as a class member here? Looks like it is not being used except from passing it to uimanager. Same applies in NativeProxy implementation for Android

Reply:
We need to store it because React Native's `UIManager` keeps a vector of raw pointers (see [here](https://github.com/facebook/react-native/blob/1a1e3994666e1f8b2abf2c5040ac3a44149ddca0/packages/react-native/ReactCommon/react/renderer/uimanager/UIManager.h#L218)).